openbb_platform/providers/tradingeconomics/openbb_tradingeconomics/models/economic_calendar.py

```diff
@@ -130,7 +198,8 @@ async def aextract_data(
     ) -> Union[dict, List[dict]]:
         """Return the raw data from the TE endpoint."""
         api_key = credentials.get("tradingeconomics_api_key") if credentials else ""
-
+        if query.group is not None:
```

Comment:
It should be enough to do `if query.group`

Reply:
PEP8 recommends always using `if foo is not None` for testing when a variable that defaults to None was set to another value.

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.
Also, beware of writing if x when you really mean if x is not None – e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

Source: https://peps.python.org/pep-0008/#programming-recommendations